### PR TITLE
New DTH for our SP2600ZB and SP2610ZB

### PR DESCRIPTION
**Pull request by SmartThings**
@shrakuma

### CODE REVIEW

File: devicetypes/sinope-technologies/sp2600zb-sp2610zb-sinope-outlet.src/sp2600zb-sp2610zb-sinope-outlet.groovy [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    definition (name: "SP2600ZB-SP2610ZB Sinope Outlet", namespace: "Sinope Technologies", author: "Sinope Technologies",  ocfDeviceType: "oic.d.smartplug") 
+    {
+        capability "Actuator"
+        capability "Configuration"
+        capability "Refresh"
+        capability "Switch"
+		capability "Health Check"
+        
+
+        fingerprint manufacturer: "Sinope Technologies", model: "SP2600ZB", deviceJoinName: "Sinope SP2600ZB Smart Plug"
```
Comment (PKacprowiczS) [inline devicetypes/sinope-technologies/sp2600zb-sp2610zb-sinope-outlet.src/sp2600zb-sp2610zb-sinope-outlet.groovy:23]: Please change definition of name to `deviceJoinName: "Sinope Outlet"` in both fingerprints

File: devicetypes/sinope-technologies/sp2600zb-sp2610zb-sinope-outlet.src/sp2600zb-sp2610zb-sinope-outlet.groovy [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+**/
+
+metadata {
+    preferences {
+        input("trace", "bool", title: "Trace", description: "Set it to true to enable tracing")
+    }
+
+    definition (name: "SP2600ZB-SP2610ZB Sinope Outlet", namespace: "Sinope Technologies", author: "Sinope Technologies",  ocfDeviceType: "oic.d.smartplug")
```
Comment (PKacprowiczS) [inline devicetypes/sinope-technologies/sp2600zb-sp2610zb-sinope-outlet.src/sp2600zb-sp2610zb-sinope-outlet.groovy:14]: There's no need for separate DTH in this case. Please move devices to existing `devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy`

**Review by PKacprowiczS — CHANGES_REQUESTED** [review]
@Sinopetech Please remove file `devicetypes/sinope-technologies/sp2600zb-sp2610zb-sinope-outlet.src/sp2600zb-sp2610zb-sinope-outlet.groovy` from this PR and add both fingerprints to
`devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy`. Devices should be working without further changes, but include them, in case they'd be necessary. Keep in mind deviceJoinName change I've mentioned in inline comment.

**Comment by PKacprowiczS** [comment]
@Sinopetech let me know if you need any assistance

**Comment by Sinopetech** [comment]
@PKacprowiczS, we've created a new pull request which contains the modification that you have suggested #29913

File: devicetypes/sinope-technologies/sp2600zb-sp2610zb-sinope-outlet.src/sp2600zb-sp2610zb-sinope-outlet.groovy (hunk 2 above)
Comment (pagejess009) [reply devicetypes/sinope-technologies/sp2600zb-sp2610zb-sinope-outlet.src/sp2600zb-sp2610zb-sinope-outlet.groovy:14]: Groovy bady

**Review by pagejess009 — COMMENTED** [review]
DTF!!! New way if you like DTF, tell me what ya all think ok yes ok let's do this who is having a hoot? Eh

**Pull request closed without merging**